cypress/support/step-definitions/dialog-steps.js

```diff
@@ -37,9 +36,23 @@ Then('stickyFormFooter is disabled', () => {
 });
 
 When('I click on {string} outside dialog', (position) => {
-  cy.iFrame(STORY_ROOT).click(position, { force: true });
+  storyRoot().click(position, { force: true });
 });
 
 When('I click on background {string} outside dialog', (position) => {
   backgroundUILocator().click(position, { force: true });
 });
+
+Then('footer buttons have color {string} and has {int} px border', (color, px) => {
+  dialogStickyFormFooter().children().eq(0).children()
```

Comment:
Maybe better will add new variable e.g 'footerButtons' with `dialogStickyFormFooter().children().eq(0).children()` and use it here. It'll be more readable

Reply:
updated